retrofit/src/main/java/retrofit/BuiltInConverters.java

```diff
@@ -25,21 +25,40 @@
 import static retrofit.Utils.closeQuietly;
 
 final class BuiltInConverters extends Converter.Factory {
+  private volatile OkHttpResponseBodyConverter streamingResponseBodyConverter;
```

Comment:
Sure. Worst case (which is practically impossible) would be every endpoint called on a different thread at the same time which would each create their own instance, but that's no worse than the current behavior which does that anyway!